docs/hpc/about/pricing-compute.md

```diff
@@ -1,33 +1,52 @@
+
 ## Pricing for Graphics Processing Unit (GPU)
 
 <div class="alert alert-info" role="alert" markdown="1">
 {{ site.cluster.name }} has {{ site.data.specs.gpu_nodes }} GPU nodes with a total of {{ site.data.specs.gpus }} GPUs available to all users. Among these, {{ site.data.specs.gpu_nodes | minus: site.data.specs.communal_gpu_nodes }} GPU nodes, with a total of {{ site.data.specs.gpus | minus: site.data.specs.communal_gpus }} GPUs, were contributed by different research groups. GPU jobs are limited to 2 hours in length when run on GPUs not contributed by the running user's lab.  In contrast, [contributors are _not_ limited to 2-hour GPU jobs on nodes they contributed](/hpc/scheduler/queues.html). There is also one GPU development node that is available to all users.
 </div>
 
-* CPU: 1 × AMD EPYC 7543P 2.8 GHz 32-core processor
-* GPU: 4 × Nvidia A40
-* RAM: 512 GiB/node
+Single GPU cards cannot be purchased - only a full GPU node. However, just like partial compute nodes can be purchased, GPU purchase requests can also be pooled towards a full system. 
+
+**All options include the following Mercury GPU208 2U Server:** 
+CPU: (1) EPYC 7543P 2.8GHz Thirty-Two-Core Procs 
+Memory: 512GB DDR4-3200 (8x 64GB)  
+OS Drive: (1) 512GB SATA SSD  
+Storage: (1) 960GB U.2 NVMe SSD  
+
+**Price per A40 GPU node: ~$25k USD (including warranty, excluding partial tax)**
+GPU: (4) A40 48GB GDDR6
 
-**Price per GPU node:  25,412 USD**
+**Price per L40 GPU node: ~$33k USD (including warranty, excluding partial tax)**
+GPU: (4) L40 48GB GDDR6 PCIe
 
+**Price per A100 GPU node: ~$58k USD (including warranty, excluding partial tax)**
+GPU: (4) A100 80GB GDDR6 PCIe 
+
+**As of May 2023, lead times for GPUs are 4-5 months from order to delivery.** Once the hardware is received, we will schedule time to configure hardware as time allows given priority projects. Once the hardware is received, we will schedule time to configure hardware as time allows given priority projects.  
 
 ## How to purchase
 
-To purchase compute nodes or GPUs, please contact [{{ site.cluster.email_compute }}](mailto:{{ site.cluster.email_compute }}?subject=Request%20to%20purchase%20compute) to discuss the specifics (requirements, pricing, availability and current lead times).
-Single GPU cards cannot be purchased, only a full GPU node. However, just like partial compute nodes can be purchased, GPU purchase requests can also be pooled towards a full system.
+To purchase compute nodes or GPUs, please contact [{{ site.cluster.email_compute }}](mailto:{{ site.cluster.email_compute }}?subject=Request%20to%20purchase%20compute). Please include what you are considering contributing and your speedtype or COA, along with any questions. .
```

Comment:
Q. What does "COA" stand for?

Reply:
chart of accounts: UCSF's chart of accounts (COA) is the coding structure that defines operations in financial terms. The COA serves as the foundation of the general ledger. The UCSF chartstring is a 38-character coding string that consists of 8 individual "blocks", or segments, called chartfields.

Maybe we can put COA/chartstring, as those are the terms people tend to use when they send me their funding info.

Reply:
> chart of accounts:  ...

Thanks. So, basically, this information allows us to bill them/their grants.

> Maybe we can put COA/chartstring, as those are the terms people tend to use when they send me their funding info.

If it helps to saving some back-and-forth emailing, then I think it could be worth it.   Are you thinking of giving examples, or is there something formal they need to know from us in order for them creating those chartstrings to send to us?

Reply:
Finance generates the chartstring and provides it to the researchers. So far, everyone who has made a request knows what it is and if they aren't a PI, they check with their PI or finance contact to decide which chartstring or speedtype to use.